rpcs3/Emu/Cell/Modules/cellAudio.h

```diff
@@ -300,45 +303,28 @@ class audio_ringbuffer
 		return backend->IsPlaying();
 	}
 
+	void convert_to_s16(void *buf, u32 sample_cnt);
+	void commit_data(f32 *buf, u32 sample_cnt);
```

Comment:
We usually put the * next to the type, not the variable name.
Same in the line above